website/content/en/docs/concepts/scheduling.md

```diff
@@ -260,6 +260,14 @@ If they all fail, Karpenter will fail to provision the pod.
 Karpenter will backoff and retry over time.
 So if capacity becomes available, it will schedule the pod without user intervention.
 
+#### Preferred Affinities
```

Comment:
I wonder if there should just be a section on "Preferences" rather than one specifically on "Preferred Affinities". I think a lot of this behavior that you are detailing below applies generally to how Karpenter thinks about preferences. 

For more detail, see: https://github.com/kubernetes-sigs/karpenter/issues/666

Reply:
I restructured this change to add a 2nd level "Preferences" section, and moved the topics that you called out in https://github.com/kubernetes-sigs/karpenter/issues/666 as sub headings. Let me know if it doesn't match what you expected.